e2e/_suites/helm/README.md

```diff
@@ -33,14 +33,14 @@ This is an example of the optional configuration:
 
    ```shell
    # Depending on the versions used, 
-   export HELM_VERSION="2.16.3"        # Helm version: for Helm v2.x.x we have to initialise Tiller right after the k8s cluster
+   export HELM_VERSION="3.2.4"        # Helm version: for Helm v2.x.x we have to initialise Tiller right after the k8s cluster
```

Comment:
Will the backport from master might cause some issues with https://github.com/elastic/e2e-testing/pull/419?

Reply:
Nope, I'm not backporting that one, because I realised it's partially updating it. I started updating the versions in the maintenance branches thinking that they were properly set in master, which was not the case